src/Microsoft.Health.Dicom.Web/appsettings.json

```diff
@@ -101,6 +101,9 @@
       "ContainerName": "workitemcontainer"
     }
   },
+  "KeyVault": {
+    "Enabled":  false
```

Comment:
Where `Endpoint` will come from?

Reply:
This is found via reflection inside of the Azure libs. I think it will look for a property called `vaultUri` in the config

Reply:
Will this won't work in PaaS and OSS, because we set Endpoint property. You will have to change that also

Reply:
But we don't use KeyVault today for DICOM directly, right? Like in OSS we use it to store secrets, but ultimately we use environment variables whose values are from key vault:
![image](https://user-images.githubusercontent.com/2955179/167226631-6461fb84-57c5-4c76-ae5d-819c33838e86.png)

So we could simply add new properties for the key vault connection too. Similarly, we don't use key vault for the DICOM service itself today in PAAS (just indirectly)

Reply:
Oh, I see it in DICOMcast: https://github.com/microsoft/dicom-server/blob/5cbc30ca3004f10b56a6d10c3d3cb044aade824d/converter/dicom-cast/src/Microsoft.Health.DicomCast.Hosting/Program.cs#L28

But I think the pattern will still work on its own -- it just uses a different API that should contradict the othe one. And I think it's probably best we align with the new SDK instead of our own setting names, especially seeing that the APIs used in DICOMcast are deprecated.